### DISCUSSION TITLE
DS214play 32bit, I know is slow, but where I can find the driver?

### DISCUSSION THREAD

**Discussion by MrEnergy64** [bb-qq/r8152#453] [Q&A]
Hi GitHub Team,

I use the Synology DS214play, INTEL Atom CE5335 (evansport
x86), and I saw in the compatibility list, that the 2.5 USB adapter will work, but slow. But where I can find the package for the manually installation?

System OS: DSM 7.1.1-42962 Update 6

the USB adapter is connected:
|__usb5          1d6b:0003:0302 09  3.00 5000MBit/s   0mA 1IF  (Linux 3.2.101 etxhci_hcd-161118 Etron xHCI Host Controller 0000:02:00.0) hub
  |__5-1         0bda:8156:3100 00  3.20 5000MBit/s  64mA 2IFs (Realtek USB 10/100/1G/2.5G LAN 0013000001)

But what means slow? CPU usage will be 100%, or the USB 2.5G adapter is slower as the built in 1G nic?

I already downloaded the source file r8152-2.18.1-1.zip. Can I use (and when yes how) the source file instead?

Thank you for your feedback in advance.

**Comment by bb-qq** [comment]
Since Realtek has stopped supporting older kernel versions, the most recent driver available for evansport is:
https://github.com/bb-qq/r8152/releases/download/2.17.1-2/r8152-evansport-2.17.1-2_7.1.spk
